setup.py

```diff
@@ -6,7 +6,7 @@
     long_description = f.read()
 
 setup(name="pipelinewise-target-snowflake",
-      version="1.11.1",
+      version="1.11.2",
```

Comment:
Can you please make changelog and version changes in separated PR?

```
Bumping version in setup.py is an individual PR and not mixed with feature or bugfix PRs
```

Can you please also bump the minor version? Introducing a new parameter is a new feature, that requires to bump minor and not patch version.